src/app/core/snippets/footer/footer.component.html

```diff
@@ -33,14 +33,14 @@ <h5 class="hsa">Honors Study Association Confluente</h5>
         <div class="col-xs-6 col-md-3">
           <a href="mailto:info@hsaconfluente.nl">info@hsaconfluente.nl</a><br>
           <a href="https://www.facebook.com/hsaconfluentenl/">Facebook</a><br>
-          <a href="https://www.linkedin.com/company/h.s.a.-confluente/">LinkedIn</a>
+          <a href="https://www.linkedin.com/company/h.s.a.-confluente/">LinkedIn</a><br>
           <a
             href="https://calendar.google.com/calendar/embed?src=hsaconfluente%40hsaconfluente.nl&ctz=Europe%2FAmsterdam">Calendar</a>
         </div>
 
         <div class="col-xs-12 col-md-6 text-right">
-          <p>This site is maintained by the <span class="hsa">H.S.A Confluente</span> <a
-            href="mailto:web@hsaconfluente.nl">Web Committee</a>.<br>
+          <p>This site is maintained by the <span class="hsa">H.S.A Confluente</span><a
+            href="mailto:web@hsaconfluente.nl"> Web Committee</a>.<br>
```

Comment:
Why this change? Older formatting seems more logical?

Reply:
nope, the original version did not actually place a space :P